universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -0,0 +1,26 @@
+package services.applicant;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.jayway.jsonpath.DocumentContext;
+
+public class ApplicantData {
+
+  private DocumentContext jsonData;
+
+  public ApplicantData(DocumentContext jsonData) {
+    this.jsonData = checkNotNull(jsonData);
+  }
+
+  public void put(String path, String key, String value) {
+    this.jsonData.put(path, key, value);
+  }
+
+  public <T> T read(String path, Class<T> type) {
```

Comment:
Suggestion (non-blocking): recommend adding `@Deprecated` annotation and comment to `read` and `put` to make it clear that this not the intended interface for this class.

Reply:
Good idea. Done.